cmd/sys-mgmt-agent/main.go

```diff
@@ -56,7 +56,7 @@ func main() {
 	flag.Usage = usage.HelpCallback
 	flag.Parse()
 	// [1] Removed two lines of code above (having to do with accepting the "--consul" aka "--c" flag) because the SMA is
-	// designed to operate independently, in a stand-alone fashion. In particular, the SMA should not rely on Consul,
+	// designed to operate independently,d in a stand-alone fashion. In particular, the SMA should not rely on Consul,
```

Comment:
I just noticed the typo here, but what is the purpose of this comment in the code? It doesn't appear to be describing anything in the code right here so I don't think it's necessary. If you choose to merge it in with this code here, please file an issue for a followup PR to remove such comments